paths/graphql/03-first-query.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Build a Query
+permalink: /graphql/first-query
+next-page: /graphql/query-info
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+ In the last section, we used the default **query** to display our username in the Results pane. Then, we added `avatarUrl` to your query which provided a link to your GitHub avatar. To learn more about what a query is, click "Tell me why" under this next exercise.
```

Comment:
I think we should call `avatarUrl` what it is by name (field? node? edge?) 👀 @allthedoll